.env.example

```
@@ -48,6 +48,18 @@ BUILDER_MODEL=
 POSTHOG_HOST=
+# Content moderation model override (default: anthropic/claude-3-5-haiku-20241022)
+CONTENT_MODERATION_MODEL=
+
+# Generation pipeline knobs (all optional — sensible defaults applied)
+# - GENERATION_TTL_HOURS: TTL after terminal status (default 24)
+# - GENERATION_EXECUTOR_TIMEOUT_MS: per-pipeline in-process timeout (default 300000 = 5 min)
+# - GENERATION_STUCK_SWEEP_THRESHOLD_MS: stuck-row sweep cutoff for crashed
+#   processes (default 600000 = 10 min; must be > EXECUTOR_TIMEOUT_MS so the
+#   in-process timer always wins under normal operation)
+GENERATION_TTL_HOURS=
+GENERATION_EXECUTOR_TIMEOUT_MS=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GENERATION_EXECUTOR_TIMEOUT_MS key should go before the GENERATION_TTL_HOURS key

(UnorderedKey)